src/tinify.rs

```diff
@@ -1,5 +1,5 @@
 use crate::client::Client;
```

Comment:
```suggestion
pub use crate::client::Client;
```

Possible change to make Client public as well if desired.